### PR TITLE
Fix pandas api exposure

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

**Comment by cpcloud** [comment]
Merging on green.

File: ibis/pandas/client.py [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 
 try:
-    infer_dtype = pd.lib.infer_dtype
-except AttributeError:
     infer_dtype = pd.api.types.infer_dtype
+except AttributeError:
+    infer_dtype = pd.lib.infer_dtype
```
Comment (jreback) [inline ibis/pandas/client.py]: maybe u want to use pandas-compat for this?

Comment (cpcloud) [inline ibis/pandas/client.py]: Oh does that exist now? If so, absolutely.

Comment (cpcloud) [inline ibis/pandas/client.py]: I'll open a new issue.

**Pull request closed without merging**